daprdocs/content/en/operations/components/setup-secret-store/supported-secret-stores/azure-keyvault.md

```diff
@@ -31,6 +31,8 @@ spec:
     value: "[your_service_principal_tenant_id]"
   - name: spnClientId
     value: "[your_service_principal_app_id]"
+  - name: spnCertificate
+    value : "[pfx_certificate_contents_k8s_secret_name]"
```

Comment:
Is `spnCertificate` specific to Kubernetes, or could it be used on other platforms given the certificate data is provided in the correct format? If so I'd say:
```suggestion
    value : "[pfx_certificate_contents]"
```